src/elements/ngm-cam-configuration.ts

```diff
@@ -253,12 +264,35 @@ export class NgmCamConfiguration extends LitElementI18n {
     }
   }
 
+  onClose() {
+    this.dispatchEvent(new CustomEvent('close'));
+    if (this.lockType) {
+      // keep commented tags to have translations
+      // i18next.t('cam_lock_info_elevation') i18next.t('cam_lock_info_angle')
+      // i18next.t('cam_lock_info_pitch') i18next.t('cam_lock_info_move')
+      showSnackbarInfo(i18next.t(`cam_lock_info_${this.lockType}`),
```

Comment:
Instead of relying on comments I would rather have an explicit switch:
```js
switch (this.lockType):
case 'angle':
  tr = i18next.t('cam_lock_info_angle');
  break;
...
```

Reply:
Do we want translation to change when the language change? If so we should let  `showSnackbarInfo` call i18next.t.

Reply:
replaced with a switch. 

To call i18next from `showSnackbarInfo` and make it work we should have some big switch with all messages, not sure that it a good way